packages/engine-ui/src/components/forms/eventHandlers.js

```diff
@@ -22,11 +22,11 @@ const handleContactInputChange = (
   }))
 }
 
-const handleDropdownSelect = (propertyName, callback) => ({
-  name,
-  lon,
-  lat,
-  county,
+const handleAddressInput = (propertyName, callback) => ({
+  name = '',
+  lon = '',
```

Comment:
lon/lat defaulted to string, but it is a number?

Reply:
Now number or undefined